js_modules/dagster-ui/packages/ui-core/src/assets/AssetWipeDialog.tsx

```diff
@@ -1,75 +1,186 @@
 import {RefetchQueriesFunction, gql, useMutation} from '@apollo/client';
-import {Button, Dialog, DialogBody, DialogFooter, Group} from '@dagster-io/ui-components';
+// eslint-disable-next-line no-restricted-imports
+import {ProgressBar} from '@blueprintjs/core';
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  Group,
+  ifPlural,
+} from '@dagster-io/ui-components';
+import {useVirtualizer} from '@tanstack/react-virtual';
+import {memo, useLayoutEffect, useMemo, useRef, useState} from 'react';
 
 import {asAssetKeyInput} from './asInput';
 import {AssetWipeMutation, AssetWipeMutationVariables} from './types/AssetWipeDialog.types';
+import {showCustomAlert} from '../app/CustomAlertProvider';
 import {PYTHON_ERROR_FRAGMENT} from '../app/PythonErrorFragment';
+import {PythonErrorInfo} from '../app/PythonErrorInfo';
 import {displayNameForAssetKey} from '../asset-graph/Utils';
+import {NavigationBlock} from '../runs/NavigationBlock';
+import {Inner, Row} from '../ui/VirtualizedTable';
+import {numberFormatter} from '../ui/formatters';
 
 interface AssetKey {
   path: string[];
 }
 
-export const AssetWipeDialog = ({
+const CHUNK_SIZE = 100;
```

Comment:
Not sure if we should have a higher chunk size here?

Reply:
i think conservatively small is the right place to start given these are writes